client/js/screens/Event/Event.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import {View, Image, Text, ScrollView, Button} from 'react-native';
+import ButtonDefault from '../../components/ButtonDefault';
+import RatingScore from '../../components/RatingScore';
+import {calculateRatingScore} from '../../lib/calculateRatingScore';
+import styles from './styles';
+import SafeAreaView from 'react-native-safe-area-view';
+import MapView from 'react-native-maps';
+import Reviews from '../../components/Reviews';
+
+const Event = ({navigation, eventInfo}) => {
+  return (
+    <>
+      <ScrollView>
+        <SafeAreaView>
+          <View style={styles.eventImg}>
+            <Image style={styles.eventImg} source={{uri: eventInfo.image}} />
+
+            <View style={styles.overlay}>
+              <Text style={styles.eventName}>{eventInfo.title}</Text>
+              <Text style={styles.eventLocation}>
+                {eventInfo.locationTitle}
+              </Text>
+              <View style={styles.eventButtons}>
+                <View style={styles.eventStarRating}>
+                  <RatingScore score={4} />
+                </View>
+                <View style={styles.eventshareButton}>
+                  <Image
+                    source={require('../../assets/images/icFaveShareDefault.png')}
+                  />
+                </View>
+                <View style={styles.eventButton}>
+                  <Image
+                    source={require('../../assets/images/icFaveCircleDefault.png')}
+                  />
+                </View>
+              </View>
+            </View>
+          </View>
+
+          <View style={styles.eventScreenInfo}>
+            <Text style={styles.infoText}>Date / Time</Text>
+            <View
+              style={{
+                ...styles.eventTextInput,
+                height: 36,
+                flexDirection: 'row',
+                alignItems: 'center',
```

Comment:
Please move the inline styles to the styles.js.

Reply:
Complete.